x/auction/types/params.go

```diff
@@ -70,16 +81,24 @@ func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 
 // Validate checks that the parameters have valid values.
 func (p Params) Validate() error {
-	if err := validateBidDurationParam(p.BidDuration); err != nil {
+	if err := validateBidDurationParam(p.ForwardBidDuration); err != nil {
+		return err
+	}
+
+	if err := validateBidDurationParam(p.ReverseBidDuration); err != nil {
 		return err
 	}
 
 	if err := validateMaxAuctionDurationParam(p.MaxAuctionDuration); err != nil {
 		return err
 	}
 
-	if p.BidDuration > p.MaxAuctionDuration {
-		return errors.New("bid duration param cannot be larger than max auction duration")
+	if p.ForwardBidDuration > p.MaxAuctionDuration {
+		return errors.New("forward bid duration param cannot be larger than max auction duration")
+	}
+
+	if p.ReverseBidDuration > p.MaxAuctionDuration {
```

Comment:
do we need to validate forward duration is more than reverse? I don't remember if that was a requirement or not.

Reply:
No need - it's a valid config to have forward > reverse